modules/@apostrophecms/i18n/i18n/en.json

```diff
@@ -173,6 +173,8 @@
   "loginPageBadCredentials": "Your credentials are incorrect, or there is no such user",
   "loginPageTitle": "Login",
   "loginRequired": "Login Required",
+  "loginMaxAttemptsReached": "Too many login attempts. You may try again in a minute.",
```

Comment:
Suggest you remove the word "login" from the translation (the English) here so it makes sense no matter whether it is the initial login or a late requirement. It's fine to have it in the key.

Reply:
Requirements are still in the login process but why not, done.